charts/catalog/templates/apiregistration.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.useAggregator }}
-apiVersion: apiregistration.k8s.io/v1alpha1
+apiVersion: apiregistration.k8s.io/v1beta1
```

Comment:
@arschles @krancour is there some helm trick we can pull for checking for kubernetes v1.6 vs v1.7 and setting to alpha or beta?